### PR TITLE
OSDOCS-3454 - Adding ROSA UI cluster creation procedures

### PR DESCRIPTION
**Pull request by pneedle-rh**
**Please do not merge this PR**

This applies to `main`, `enterprise-4.11` and `enterprise-4.10`.

This relates to https://issues.redhat.com/browse/OSDOCS-3454, https://issues.redhat.com/browse/OSDOCS-3496 and https://issues.redhat.com/browse/OSDOCS-3204. The PR adds ROSA UI create cluster content to the ROSA getting started assembly, as well as to the main STS cluster creation assemblies.

The previews are as follows:

* **Getting started** -> **Getting started with ROSA** -> [**Creating a ROSA cluster with STS**](https://deploy-preview-44081--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-getting-started.html#rosa-getting-started-creating-a-cluster)

* **Installing, accessing, and deleting ROSA clusters** -> **Creating a ROSA cluster with STS using the default options** -> [**Creating a cluster with the default options through OpenShift Cluster Manager**](https://deploy-preview-44081--osdocs.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html#rosa-sts-creating-cluster-using-defaults-ocm_rosa-sts-creating-a-cluster-quickly)

* **Installing, accessing, and deleting ROSA clusters** -> **Creating a ROSA cluster with STS using customizations** -> **[Understanding the auto and manual deployment modes](https://deploy-preview-44081--osdocs.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-understanding-deployment-modes_rosa-sts-creating-a-cluster-with-customizations)**

* **Installing, accessing, and deleting ROSA clusters** -> **Creating a ROSA cluster with STS using customizations** -> **Understanding the auto and manual deployment modes** -> **[Creating the Operator roles and OIDC provider through OpenShift Cluster Manager](https://deploy-preview-44081--osdocs.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-creating-operator-roles-and-oidc-manually-ocm_rosa-sts-creating-a-cluster-with-customizations)**

* **Introduction to ROSA** -> **About IAM resources for ROSA with STS** -> **Cluster-specific Operator IAM role reference** -> [**Defining an Operator IAM role prefix**](https://deploy-preview-44081--osdocs.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-defining-an-operator-role-prefix_rosa-sts-about-iam-resources)

### CODE REVIEW

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
++
+[source,terminal]
+----
+$ rosa create account-roles --mode manual <1>
+----
+<1> `manual` mode generates the `aws` CLI commands and JSON files needed to create the account-wide roles and policies. After review, you must run the commands manually to create the resources.
++
+.. After review, run the `aws` commands manually to create the roles and policies. Alternatively, you can run the preceding command using `--mode auto` to run the `aws` commands immediately.
+
+. (Optional) If you are using your own AWS KMS key to encrypt the control plane data volumes and the persistent volumes (PVs) for your applications, add the ARN for the account-wide installer role to your KMS key policy.
```
Comment (bmcelvee) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:52]: This is existing content, but I noticed it while reading through:
s/(Optional)/Optional:

Comment (pneedle-rh) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:52]: @bmcelvee thanks for highlighting this. Resolved.

**Comment by pneedle-rh** [comment]
@0kashi, @arendej, @dtaylor113, @xueli181114, @yuwang-RH and @pvasant can you please review this docs pull request? Thanks!

**Comment by pneedle-rh** [comment]
Due to "bad ref" remote rejection errors when trying to push updates to the fork branch for this PR, I am closing the PR. I will either reopen it or provide an alternative PR link shortly.

**Comment by pneedle-rh** [comment]
This content is now being applied through https://github.com/openshift/openshift-docs/pull/45306.

**Pull request closed without merging**